oe_theme.theme

```diff
@@ -418,6 +420,27 @@ function oe_theme_preprocess_pattern_dialog(array &$variables): void {
   $variables['hidden'] = $variables['hidden'] ? 'true' : 'false';
 }
 
+/**
+ * Implements hook_preprocess_HOOK() for pattern gallery template.
+ */
+function oe_theme_preprocess_pattern_gallery(array &$variables): void {
```

Comment:
Are we sure this is the approach we want to give the gallery pattern? This makes its hard for users of the theme to have galleries of different sizes without overriding the theme.

Reply:
I would keep this one in the scope of this ticket as we have no input from business, we can use the demo to gather input from them in this respect.

Reply:
:+1: